### PR TITLE
feat(kafka): avro schema required; delete deprecated outbound template

### PR DESCRIPTION
**Pull request by Oleksiivanov**
## Description

Fixed the Kafka template: Avro schema is now required;
 deprecated Kafka outbound template has been removed.

## Related issues
https://github.com/camunda/team-connectors/issues/639

### CODE REVIEW

File: connectors/kafka/element-templates/kafka-outbound-connector-hybrid.json [hunk 1]
```diff
@@ -174,8 +174,11 @@
   }, {
     "id" : "avro.schema",
     "label" : "Avro schema",
-    "description" : "Optional Avro schema for the message value",
-    "optional" : true,
+    "description" : "Avro schema for the message value",
+    "optional" : false,
```
Comment (johnBgood) [inline connectors/kafka/element-templates/kafka-outbound-connector-hybrid.json:178]: isn't `false` the default value? If so, we might remove this line I guess

File: connectors/kafka/element-templates/kafka-outbound-connector.json [hunk 2]
```diff
@@ -169,8 +169,11 @@
   }, {
     "id" : "avro.schema",
     "label" : "Avro schema",
-    "description" : "Optional Avro schema for the message value",
-    "optional" : true,
+    "description" : "Avro schema for the message value",
+    "optional" : false,
```
Comment (johnBgood) [inline connectors/kafka/element-templates/kafka-outbound-connector.json:173]: same here

**Review by johnBgood — APPROVED** [review]
👍

File: connectors/kafka/element-templates/kafka-outbound-connector-hybrid.json (hunk 1 above)
Comment (johnBgood) [reply connectors/kafka/element-templates/kafka-outbound-connector-hybrid.json:178]: Ha ok, I resolve the conv then sorry about that

Comment (sbuettner) [reply connectors/kafka/element-templates/kafka-outbound-connector-hybrid.json:178]: Great feedback @johnBgood 
@Oleksiivanov Thanks for the clarification. Can you please add this feedback to this ticket please: https://github.com/camunda/connectors/issues/1797

**Pull request merged**